### ISSUE TITLE
Fix color contrast issues associated with primary accent color

### ISSUE THREAD

**Issue by schne324** [dequelabs/workshop-08-2024#1]
## axe Issues

- https://axe.deque.com/issues/02069513-4d93-4512-a693-b642ee830453

## Designer Notes

Use `#054A91` instead to increase the contrast ratio and become WCAG 2.1 AA compliant

## Dev Notes

> [!NOTE]
> You should be able to fix this with just 1 line of CSS change -- see `fictitious.css`
